MakieCore/src/recipes.jl

```diff
@@ -678,7 +678,10 @@ function Base.showerror(io::IO, i::InvalidAttributeError)
 end
 
 function attribute_name_allowlist()
-    (:xautolimits, :yautolimits, :zautolimits, :label, :rasterize, :model, :transformation)
+    (
+        :xautolimits, :yautolimits, :zautolimits, :label, :rasterize, :model,
+        :transformation, :inspector_label
```

Comment:
I think inspector_label should go into general plot attributes. As I understand it, every plot should be able to have that, and it should be documented as part of the plot docs.